src/PublishPort.jsx

```diff
@@ -36,7 +36,7 @@ export const PublishPort = ({ id, item, onChange, idx, removeitem, itemCount })
                         <Popover aria-label={_("Host port help")}
                             enableFlip
                             bodyContent={_("If the host port is not set the container port will be randomly assigned a port on the host.")}>
-                            <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                            <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1299-20230601-052642-7ca6cf83-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)